src/analyticalSolutions/compare_solutions.py

```diff
@@ -0,0 +1,304 @@
+        exit(-1)
+    else:
+        print("Reading Solution   file [" + solFile + "  ] ...")
+        file  = open(solFile, 'r')
+        
+        # Read data lines without header
+        lines = file.readlines()[1:]
+
+        # Empty vectors
+        sol_r = []
+        sol_rho = []
+        sol_u = []
+        sol_p = []
+        sol_vel = []
+        sol_cs = []
+        sol_rhoShock = []
+        sol_uShock = []
+        sol_pShock = []
+        sol_velShock = []
+        sol_csShock = []
+        sol_rho0 = []
+        
+        for line in lines:
+            tokens = line.split()
+            
+            sol_r.append(        float(tokens[ 0]) )
+            sol_rho.append(      float(tokens[ 1]) )
+            sol_u.append(        float(tokens[ 2]) )
+            sol_p.append(        float(tokens[ 3]) )
+            sol_vel.append(      float(tokens[ 4]) )
+            sol_cs.append(       float(tokens[ 5]) )
+            sol_rhoShock.append( float(tokens[ 6]) )
+            sol_uShock.append(   float(tokens[ 7]) )
+            sol_pShock.append(   float(tokens[ 8]) )
+            sol_velShock.append( float(tokens[ 9]) )
+            sol_csShock.append(  float(tokens[10]) )
+            sol_rho0.append(     float(tokens[11]) )
+            
+        file.close()
+            
+    # Load Simulated file
+    if (not os.path.isfile(simFile)):
+        print("Simulation file [" + simFile + "] doesn't exist.")
+        exit(-1)
+    else:
+        print("Reading Simulation file [" + simFile + "] ...")
+        file  = open(simFile, 'r')
+        
+        # Read data lines without header
+        lines = file.readlines()[1:]
+
+        # Empty vectors
+        sim_r = []
+        sim_rho = []
+        sim_u = []
+        sim_p = []
+        sim_vel = []
+        sim_cs = []
+        sim_rhoShock = []
+        sim_uShock = []
+        sim_pShock = []
+        sim_velShock = []
+        sim_csShock = []
+        sim_rho0 = []
+        
+        for line in lines:
+            tokens = line.split()
+            
+            sim_r.append(        float(tokens[ 0]) )
+            sim_rho.append(      float(tokens[ 1]) )
+            sim_u.append(        float(tokens[ 2]) )
+            sim_p.append(        float(tokens[ 3]) )
+            sim_vel.append(      float(tokens[ 4]) )
+            sim_cs.append(       float(tokens[ 5]) )
+            sim_rhoShock.append( float(tokens[ 6]) )
+            sim_uShock.append(   float(tokens[ 7]) )
+            sim_pShock.append(   float(tokens[ 8]) )
+            sim_velShock.append( float(tokens[ 9]) )
+            sim_csShock.append(  float(tokens[10]) )
+            sim_rho0.append(     float(tokens[11]) )
+            
+        file.close()
+        
+    
+
+    # Plot graphics
+    if (not no_plots):
+        
+        print("\nGenerating graphics ...")
+        
+        figureName = "./sedov_density_comparation_"   + time.__str__() + ".svg"
```

Comment:
comparison

Reply:
also here, perhaps we can just use `sedov_density_` for brevity.